### DISCUSSION TITLE
Wonderswan Adapter

### DISCUSSION THREAD

**Discussion by RagingKraken** [sanni/cartreader#545] [Need help]
This is probably an easy question, but I've built an adapter before and have only done some basic soldering when fixing up consoles. I have a Wonderswan PCB adaptor for the SNES slot. Obviously, I'm going to have to solder the cartridge slot  to the PCB, but that's where I'm kind of stuck. Following the WS adapter v2 that was shared here, all I have to do is solder the slot onto the PCB right? I'm not quite sure what to use the HW 3/4 pads for. I'm on HW3 right now.

Also, does anyone know where to source the cartridge slots? I've had no luck finding a new/replacement one, so I might have to go to my last resort of cannibalizing an actual WonderSwan.

**Comment by PsyK0p4T** [comment]
That's what I did. Took a black & white model, they are pretty cheap. And auctions sites are full of defect units.
I salvaged the cartridge slot from one and used it for my adapter.

**Comment by splash5** [comment]
If you are using HW3, then close solder pad 3 (o-o o), if you are using HW4 or above, close pad 4 (o o-o).

**Reply by RagingKraken** [reply]
Thanks for that heads up. I'll remember to do that when I get around to pulling the cartridge slot from my spare console.
